runtimes/qf-runtime/src/configs/mod.rs

```diff
@@ -103,6 +109,126 @@ impl pallet_aura::Config for Runtime {
     type SlotDuration = pallet_aura::MinimumPeriodTimesTwo<Runtime>;
 }
 
+parameter_types! {
+    pub const Period: BlockNumber = 1;
+    pub const Offset: BlockNumber = 0;
+}
+
+impl pallet_session::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type ValidatorId = <Self as frame_system::Config>::AccountId;
+    type ValidatorIdOf = pallet_staking::StashOf<Self>;
+    type ShouldEndSession = pallet_session::PeriodicSessions<Period, Offset>;
+    type NextSessionRotation = pallet_session::PeriodicSessions<Period, Offset>;
+    type SessionHandler = <SessionKeys as OpaqueKeys>::KeyTypeIdProviders;
+    type Keys = SessionKeys;
+    type DisablingStrategy = pallet_session::disabling::UpToLimitWithReEnablingDisablingStrategy;
+
+    type WeightInfo = pallet_session::weights::SubstrateWeight<Runtime>;
+    type SessionManager = Staking;
+}
+
+impl pallet_session::historical::Config for Runtime {
+    type FullIdentification = ();
+    type FullIdentificationOf = pallet_staking::NullIdentity;
+}
+
+pallet_staking_reward_curve::build! {
+    const REWARD_CURVE: PiecewiseLinear<'static> = curve!(
+        min_inflation: 0_025_000,
+        max_inflation: 0_100_000,
+        ideal_stake: 0_500_000,
+        falloff: 0_050_000,
+        max_piece_count: 40,
+        test_precision: 0_005_000,
+    );
+}
+
+parameter_types! {
+    pub const MaxElectingVoters: u32 = 22_500;
+    /// We take the top 22500 nominators as electing voters and all of the validators as electable
+    /// targets. Whilst this is the case, we cannot and shall not increase the size of the
+    /// validator intentions.
+    pub ElectionBounds: frame_election_provider_support::bounds::ElectionBounds =
+        ElectionBoundsBuilder::default().voters_count(MaxElectingVoters::get().into()).build();
+    // Maximum winners that can be chosen as active validators
+    pub const MaxActiveValidators: u32 = 1000;
+    // One page only, fill the whole page with the `MaxActiveValidators`.
+    pub const MaxWinnersPerPage: u32 = MaxActiveValidators::get();
+    // Unbonded, thus the max backers per winner maps to the max electing voters limit.
+    pub const MaxBackersPerWinner: u32 = MaxElectingVoters::get();
+}
+
+pub type OnChainAccuracy = sp_runtime::Perbill;
+
+pub struct OnChainSeqPhragmen;
+impl onchain::Config for OnChainSeqPhragmen {
+    type Sort = ConstBool<true>;
+    type System = Runtime;
+    type Solver = SequentialPhragmen<AccountId, OnChainAccuracy>;
+    type DataProvider = Staking;
+    type WeightInfo = frame_election_provider_support::weights::SubstrateWeight<Runtime>;
+    type Bounds = ElectionBounds;
+    type MaxBackersPerWinner = MaxBackersPerWinner;
+    type MaxWinnersPerPage = MaxWinnersPerPage;
+}
+
+parameter_types! {
+    pub const SessionsPerEra: sp_staking::SessionIndex = 6;
+    pub const BondingDuration: sp_staking::EraIndex = 24 * 28;
```

Comment:
Let's use default values from the pallet to allow quick unstaking in just three eras (9 minutes with 1 minute long sessions?).

```suggestion
    pub const SessionsPerEra: sp_staking::SessionIndex = 3;
    pub const BondingDuration: sp_staking::EraIndex = 3;
```

Reply:
Done